site2/docs/client-libraries-go.md

```diff
@@ -16,18 +16,35 @@ You can use Pulsar [Go client](https://github.com/apache/pulsar-client-go) to cr
 
 ### Install go package
 
-You can install the `pulsar` library locally using `go get`.  
+You can install the `pulsar` library locally using `go get` download to local or downloading online.  
+
+Download to local:
 
 ```bash
 $ go get -u "github.com/apache/pulsar-client-go/pulsar"
 ```
+If you can't download, please try it by configure [CDN accelerated proxy]( https://learnku.com/go/wikis/38122).
```

Comment:
this should only applied to China users due the network issue, so i dont think it is proper to put it here.